src/components/DopplerPlus/UpgradeSuggestionForm.test.js

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import { render, cleanup, fireEvent, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import UpgradeSuggestionForm from './UpgradeSuggestionForm';
+import { AppServicesProvider } from '../../services/pure-di';
+import DopplerIntlProvider from '../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+import { act } from 'react-dom/test-utils';
+
+describe('UpgradeSuggestionForm component', () => {
+  afterEach(cleanup);
```

Comment:
it is not necessary to do the cleanup, jest and mocha already do it automatically